packages/reactive-element/src/reactive-element.ts

```diff
@@ -817,6 +821,9 @@ export abstract class ReactiveElement
   }
 
   /**
+   * Registers a `ReactiveController` to participate in the element's reactive update cycle.
+   * The element automatically calls into any registered controllers during its
+   * lifecycle callbacks.
```

Comment:
Add "If the element is connected when `addController()` is called, the controller's `hostConnected()` callback will be immediately called.